independent-publisher-connectors/Electricity Maps/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1239 @@
+                      "description": "Geothermal power production in MW",
+                      "x-ms-summary": "Geothermal"
+                    },
+                    "hydro": {
+                      "type": "number",
+                      "description": "Hydro power production in MW",
+                      "x-ms-summary": "Hydro"
+                    },
+                    "hydro discharge": {
+                      "type": "number",
+                      "description": "Hydro discharge production in MW",
+                      "x-ms-summary": "Hydro discharge"
+                    },
+                    "nuclear": {
+                      "type": "number",
+                      "description": "Nuclear power production in MW",
+                      "x-ms-summary": "Nuclear"
+                    },
+                    "oil": {
+                      "type": "number",
+                      "description": "Oil power production in MW",
+                      "x-ms-summary": "Oil"
+                    },
+                    "solar": {
+                      "type": "number",
+                      "description": "Solar power production in MW",
+                      "x-ms-summary": "Solar"
+                    },
+                    "unknown": {
+                      "type": "number",
+                      "description": "Unknown power production in MW",
+                      "x-ms-summary": "Unknown"
+                    },
+                    "wind": {
+                      "type": "number",
+                      "description": "Wind power production in MW",
+                      "x-ms-summary": "Wind"
+                    }
+                  },
+                  "type": "object",
+                  "description": "Power Production Data",
+                  "x-ms-summary": "Power Production"
+                },
+                "powerProductionTotal": {
+                  "type": "number",
+                  "description": "Power Production Total(in MW)",
+                  "x-ms-summary": "Total Production"
+                },
+                "renewablePercentage": {
+                  "type": "number",
+                  "description": "Percentage of renewable energy",
+                  "x-ms-summary": "Renewables Percentage"
+                },
+                "updatedAt": {
+                  "type": "string",
+                  "description": "Date and time when data has been updated",
+                  "x-ms-summary": "Updated At"
+                },
+                "zone": {
+                  "type": "string",
+                  "description": "Country or Zone",
+                  "x-ms-summary": "Zone"
+                }
+              },
+              "type": "object"
+            }
+          }
+        },
+        "operationId": "BreakdownLatest",
+        "summary": "Get Live Power Breakdown",
+        "description": "Retrieves the last known data about the origin of electricity in an area"
```

Comment:
@VitaliiSorokin all operations descriptions should end with a period, No period required only if there's a single word. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)